### DISCUSSION TITLE
Vercel IP is blocked in Russia, Docs are not available

### DISCUSSION THREAD

**Discussion by ivansky** [vercel/next.js#31528] [Help]
Hi,

There are some sites that uses vercel as hosting contain some data that is illegal in Russia,

https://isitblockedinrussia.com/?host=nextjs.org

And because of hosting shares the same IP for client sites and main documentation site, it's impossible to read docs.

I'm already using additional VPN, but it's kind of a problem because usually we need work-related VPN already and mix them is a pain.

Is it possible to separate main sites IP from external sites?

**Comment by merelinguist** [comment]
Could you use web.archive.org? It doesn't appear to be blocked in Russia

**Comment by liaohongxing** [comment]
use clash  access 

https://github.com/Dreamacro/clash
